### ISSUE TITLE
Can't authenticate to Git repo with password

### ISSUE THREAD

**Issue by theandrelima** [microsoft/vscode#136970]
Issue Type: <b>Bug</b>

Out of the blue, VS Code just completely stopped prompting for username/password when I try to do anything that requires authentication (pull, push, fetch...). 

I do not have any other form of authentication set-up, like SSH keys or API tokens, so this is effectively impedinding work with VS Code. If I try it from other editors/terminal, it works perfectly fine. So this is definitely something in VSCode.

By the way, password authentication has been working normally with VSCode since I started using it about 3-4 months ago.

VS Code version: Code 1.62.1 (f4af3cbf5a99787542e2a30fe1fd37cd644cc31f, 2021-11-05T10:57:55.946Z)
OS version: Windows_NT x64 10.0.18363
Restricted Mode: No
Remote OS version: Linux x64 5.4.72-microsoft-standard-WSL2
Remote OS version: Linux x64 5.4.72-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-10310U CPU @ 1.70GHz (8 x 2208)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.61GB (17.38GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu-20.04|
|OS|Linux x64 5.4.72-microsoft-standard-WSL2|
|CPUs|Intel(R) Core(TM) i5-10310U CPU @ 1.70GHz (8 x 2208)|
|Memory (System)|24.75GB (22.30GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu-20.04|
|OS|Linux x64 5.4.72-microsoft-standard-WSL2|
|CPUs|Intel(R) Core(TM) i5-10310U CPU @ 1.70GHz (8 x 2208)|
|Memory (System)|24.75GB (22.30GB free)|
|VM|0%|
</details><details><summary>Extensions (22)</summary>

Extension|Author (truncated)|Version
---|---|---
remote-wsl|ms-|0.58.5
atlascode|atl|2.9.1
vscode-django|bat|1.6.0
python-environment-manager|don|1.0.3
python-extension-pack|don|1.7.0
gitlens|eam|11.6.1
vscode-pull-request-github|Git|0.32.0
vsc-python-indent|Kev|1.14.2
MagicPython|mag|1.1.0
git-graph|mhu|1.30.0
vscode-docker|ms-|1.17.0
python|ms-|2021.11.1422169775
vscode-pylance|ms-|2021.11.1
jupyter|ms-|2021.10.1001414422
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.3
autodocstring|njp|0.5.4
vscode-yaml|red|1.2.0
vscodeintellicode|Vis|1.2.14
gitblame|wad|8.1.0
jinja|who|0.0.8
markdown-all-in-one|yzh|3.4.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
vsccppwtct:30382698
bridge0708:30335490
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis300:30386381
vsaa593:30376534
vssld246cf:30386378

```

</details>

**Comment by gjsjohnmurray** [comment]
/duplicate of #136837 and will be fixed in upcoming 1.62.2. Until then a workaround is to disable automatic updates and install 1.62.0.

**Comment by theandrelima** [comment]
I just found out that disabling 'Git: Terminal Authentication' is also a valid work-around for those out there that also use the embedded VS Code terminal. 

Disabling that option will prompt for credentials on the terminal itself. Works for me until we have a definitive fix...
